bin/anvi-setup-kegg-kofams

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+
+import sys
+
+import anvio
+import anvio.kegg as kegg
+
+from anvio.errors import ConfigError, FilesNPathsError
+from anvio.terminal import time_program
+
+__author__ = "Developers of anvi'o (see AUTHORS.txt)"
+__copyright__ = "Copyleft 2015-2020, the Meren Lab (http://merenlab.org/)"
+__license__ = "GPL 3.0"
+__version__ = anvio.__version__
+__maintainer__ = "Iva Veseli"
+__email__ = "iveseli@uchicago.edu"
+__provides__ = ["kofam-data", "kegg-modules-db",]
```

Comment:
I think it only provides `kegg-db` (`kegg-modules-db` is too wordy). in addition, as far as I can see, we can remove `kofam-data` from every provides / requires statements, and things wold make more sense :) ignore this if you don't agree. 

Plus, please run `anvi-script-gen-programs-network` in your branch.

Reply:
Meren, I ran `anvi-script-gen-programs-network` and got a `NETWORK.json` file. Is there a specific place I should put this file in the `anvio` or `web` repository? :)

Reply:
no :) I just was just wondering whether you'd get an error for missing `kegg-modules-db` description in programs.py

Reply:
I did before, but now I have fixed that :)